IPython/core/magics/execution.py

```diff
@@ -1285,15 +1294,21 @@ def time(self,line='', cell=None, local_ns=None):
                 self.shell.showtraceback()
                 return
             end = clock2()
+        
         else:
             st = clock2()
             try:
                 exec(code, glob, local_ns)
+                out=None
+                # multi-line %%time case
+                if expr_val and isinstance(expr_val, ast.Expr):                           
+                    out = eval(code_val, glob, local_ns)  
             except:
-                self.shell.showtraceback()
-                return
+                self.shell.showtraceback()               
+                return            
```

Comment:
If I'm not mistaken this only adds extra unneeded spaces.

Reply:
Did you mean showtraceback() ? That would be for the entire try block (from line 1301 to 1305)

Reply:
What I meant was that your diff is full of spacing changes :smile:
![B95A112B-B9A9-426A-BBE7-4396C0179C04](https://user-images.githubusercontent.com/10530980/57558663-dbaafb00-737e-11e9-99fb-836a9165b0dc.jpeg)

Reply:
Oops, thank you ! Fixed that now, should be fine.